### PR TITLE
Update 'tommath' to v1.0.1

### PR DESCRIPTION
**Pull request by t0p4**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/update.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/t0p4/hunter/builds/19317949
  * https://travis-ci.com/t0p4/hunter/builds/87118037

---

### CODE REVIEW

File: cmake/projects/tommath/hunter.cmake [hunk 1]
```diff
@@ -19,7 +19,16 @@ hunter_add_version(
...
     733e4644a0b01260830a402b457ca82064fe30cd
 )
-
+hunter_add_version(
+    PACKAGE_NAME
+    tommath
+    VERSION
+    "1.0.1"
+    URL
+    "https://github.com/t0p4/libtommath/archive/v1.0.1-p1.tar.gz"
```
Comment (ruslo) [inline cmake/projects/tommath/hunter.cmake]: Please send your patches to https://github.com/hunter-packages/libtommath first.

I've created new branch `hunter-1.0.1` basing on `v1.0.1` tag, feel free to send pull request for review there, see:
* https://docs.hunter.sh/en/latest/creating-new/patch.html

**Comment by t0p4** [comment]
https://github.com/hunter-packages/libtommath/pull/4

File: cmake/projects/tommath/hunter.cmake (hunk 1 above)
Comment (ruslo) [inline cmake/projects/tommath/hunter.cmake]: Please use https://github.com/hunter-packages/libtommath/releases/tag/v1.0.1-p0 here.

Comment (t0p4) [reply cmake/projects/tommath/hunter.cmake]: have done it

File: cmake/projects/tommath/hunter.cmake [hunk 2]
```diff
@@ -19,7 +19,16 @@ hunter_add_version(
...
-
+hunter_add_version(
+    PACKAGE_NAME
+    tommath
+    VERSION
+    "1.0.1"
+    URL
+    "https://github.com/hunter-packages/libtommath/archive/v1.0.1-p0.tar.gz"
+    SHA1
+    6270FF3C125D0C513AEB10F62EF25D4EB341AF37
```
Comment (ruslo) [inline cmake/projects/tommath/hunter.cmake]: Please use lower-case.

File: cmake/projects/tommath/hunter.cmake [hunk 3]
```diff
@@ -19,7 +19,16 @@ hunter_add_version(
     SHA1
     733e4644a0b01260830a402b457ca82064fe30cd
 )
-
+hunter_add_version(
+    PACKAGE_NAME
+    tommath
+    VERSION
+    "1.0.1"
```
Comment (ruslo) [inline cmake/projects/tommath/hunter.cmake]: Please use `1.0.1-p0` here.

File: cmake/configs/default.cmake [hunk 4]
```diff
@@ -388,7 +388,7 @@ hunter_default_version(tinydir VERSION 1.2-p0)
 hunter_default_version(tinyxml2 VERSION 6.2.0-p1)
 hunter_default_version(toluapp VERSION 1.0.93-p1)
 hunter_default_version(tomcrypt VERSION 1.17-p3)
-hunter_default_version(tommath VERSION 1.0-p2)
+hunter_default_version(tommath VERSION 1.0.1)
```
Comment (ruslo) [inline cmake/configs/default.cmake]: Please use `1.0.1-p0` here.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/builds/19371498
* https://travis-ci.org/ingenue/hunter/builds/439142836?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/builds/19372632
* https://travis-ci.org/ingenue/hunter/builds/439158329?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.32

**Pull request merged**